atm.js

```diff
@@ -1,3 +1,134 @@
-$(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
-});
+//initial balance
+var CcurrentBalance = 0;
+var ScurrentBalance = 0;
+
+//changing the initial display
+$('#checking div.balance').text("$" + CcurrentBalance);
+$('#savings div.balance').text("$" + ScurrentBalance);
+
+//making it red initially
+$('#savings div.balance').addClass('zero');
+$('#checking div.balance').addClass('zero');
+
+
+//saving deposit button function
+$('#savings input.deposit').on('click', function(){
+        //retreiving deposit value
+        var depositAmt = $('#savings input.input').val();
+        console.log(depositAmt);
+
+        //changing deposit to Number
+        depositAmt = parseInt(depositAmt);
+        console.log(depositAmt);
+
+        //adding to the current balance
+        ScurrentBalance = ScurrentBalance + depositAmt;
+        console.log(ScurrentBalance);
+
+        if (ScurrentBalance > 0){
+        $('#savings div.balance').removeClass('zero');
+        }
+
+        //display the currentBalance
+        $('#savings div.balance').html(ScurrentBalance);
+})
+
+
+//saving withdraw button function
+$('#savings input.withdraw').on('click', function(){
+        //retreiving deposit value
+        var withdrawAmt = $('#savings input.input').val();
+        console.log(withdrawAmt);
+
+        //changing deposit to Number
+        withdrawAmt = parseInt(withdrawAmt);
+        console.log(withdrawAmt);
+
+        //can't go negative
+        if (withdrawAmt > ScurrentBalance){
+          alert('you are broke!')
+        } else {
+          //adding to the current balance
+          ScurrentBalance = ScurrentBalance - withdrawAmt;
+          console.log(ScurrentBalance);
+
+          //add and remove class based on the balance
+          if (ScurrentBalance > 0){
+            $('#savings div.balance').removeClass('zero');
+          } else {
+            $('#savings div.balance').addClass('zero')
+          }
+
+          //display the currentBalance
+          $('#savings div.balance').html(ScurrentBalance);
+        }
+
+
+})
+
+
+//checking deposit button function
+$('#checking input.deposit').on('click', function(){
```

Comment:
Notice that your checking and savings accounts do deposits and withdrawals the exact same way. Having a deposit function for checking and one for savings isn't too bad when you just have 2 accounts, but what if you had 200 accounts...?